apps/web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@calcom/web",
-  "version": "3.4.7",
```

Comment:
why?

Reply:
caching issues for local translations.

Reply:
This shouldn't be removed though. If we want to look at modifying https://github.com/calcom/cal.com/blob/main/packages/trpc/server/createNextApiHandler.ts#L67 for local dev, that's a potential solution.

Reply:
That's fine by me.